Mathlib/LinearAlgebra/RootSystem/Defs.lean

```diff
@@ -47,16 +51,24 @@ between roots and coroots is (implicitly) included and the coroots are included
 Empirically this seems to be by far the most convenient design and by providing extensionality
 lemmas expressing the uniqueness we expect to get the best of both worlds.
 
-A final point is that we require roots and coroots to be injections from a base indexing type `ι`
-rather than subsets of their codomains. This design was chosen to avoid the bijection between roots
-and coroots being a dependently-typed object. A third option would be to have the roots and coroots
-be subsets but to avoid having a dependently-typed bijection by defining it globally with junk value
-`0` outside of the roots and coroots. This would work but lacks the convenient symmetry that the
-chosen design enjoys: by introducing the indexing type `ι`, one does not have to pick a direction
+
+Furthermore, we require roots and coroots to be injections from a base indexing type `ι` rather than
```

Comment:
If we use semantic linebreaks, then these diffs would be much smaller and easier to review.

https://sembr.org/

If you like this idea, feel free to switch to semantic linebreaks in a future PR. (No need to change this PR.)

Reply:
Personally I don't like sembr but of course I've no objection if anyone wants us to use them here.